arrow-core/src/main/kotlin/arrow/core/extensions/ordering/semigroup/OrderingSemigroup.kt

```diff
@@ -0,0 +1,46 @@
+package arrow.core.extensions.ordering.semigroup
+
+import arrow.core.Ordering
+import arrow.core.Ordering.Companion
+import arrow.core.extensions.OrderingSemigroup
+import kotlin.PublishedApi
+import kotlin.Suppress
+import kotlin.jvm.JvmName
+
+/**
+ * cached extension
+ */
+@PublishedApi()
+internal val semigroup_singleton: OrderingSemigroup = object :
+    arrow.core.extensions.OrderingSemigroup {}
+
+@JvmName("plus")
+@Suppress(
+  "UNCHECKED_CAST",
+  "USELESS_CAST",
+  "EXTENSION_SHADOWED_BY_MEMBER",
+  "UNUSED_PARAMETER"
+)
+@Deprecated("@extension kinded projected functions are deprecated", ReplaceWith("combine(arg1)", "arrow.core.combine"))
```

Comment:
```suggestion
@Deprecated("@extension kinded projected functions are deprecated", ReplaceWith("this + arg1"))
```

There is no `combine` defined on `Ordering`